psga/config/psga-schema-sars-cov-2.yaml

```diff
@@ -2,30 +2,6 @@ pipeline:
   name: SARS CoV 2
   docker_image: "sars-cov-2-pipeline"
   protocols:
-    # unknown primer
```

Comment:
mm.. removing this, means that we are skip the feature inferring the primer when this is not known..

Reply:
We do want to keep this - this is the required for primer autodetection I believe?

Reply:
If '--kit unknown`, the detected primer is used downstream.
If '--kit ARTIC_V4` (for instance), the detected primer is still reported in the results.csv file, but the input primer is used downstream. 

Commenting the `#unknown primer` block means that we cannot process those samples for which we do not know the kit. Basically, we lose functionality.